lib/parser.js

```diff
@@ -107,7 +113,7 @@ const chunk = ({
   payload   // payload // Buffer
 }) => {
   const chunk = Buffer.alloc(CHUNK_LENGTH);
-  chunk.writeInt8(constObj.STRUCT_CHUNK);
+  chunk.writeInt8(constants.STRUCT_CHUNK);
   // Write chunk fields to buffer
```

Comment:
Why there is this comment here, but there is no comment in similar function `parcel`?

Reply:
Also, I don't think that this comment is needed here